src/Tizen.Multimedia.MediaPlayer/Player/PlayerEnums.cs

```diff
@@ -295,4 +295,32 @@ public enum StreamMetadataKey
         /// </summary>
         Year
     }
+
+    /// <summary>
+    /// Enumeration of audio extract option.
+    /// </summary>
+    /// <seealso cref="Player.SetAudioFrameDecodedCallback"/>
```

Comment:
`Player.SetAudioFrameDecodedCallback` is not public API

Reply:
i've fixed it